site/app/templates/grading/electronic/ta_status/StatusStaticAnalysis.twig

```diff
@@ -0,0 +1,149 @@
+<div id="static-analysis" class="page-content">
+    <div>
+        <label for="x-axis-selector">Select X-Axis:</label>
+        <select id="x-axis-selector">
+            <option value="runtime">Runtime</option>
+            <option value="memory">Memory</option>
+            <option value="sloc">SLOC</option>
+        </select>
+    </div>
+    <div id="myDiv4"></div>
+    <script>
```

Comment:
The logic in this script tag should be moved to an external stylesheet.  You can keep the most minimal amount of JS code necessary to inject data via Twig if you want to keep the current approach.  A better approach would be to load the data asynchronously via JS.